src/components/FilmList/FilmList.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Films from './Films';
+import { isNonEmptyArray } from './deps';
+import { FilmLists } from './type';
+
+const films: any = ['How to train your dragon', 'Fast & Furious 6', 'Gladiator', 'Captain Marvel', 'Iron Man 3'];
+const resultArray: any = [];
+
+class FilmList extends React.Component {
+  state: FilmLists = {};

```

Comment:
This should be:
`state: FilmLists = { list: [] };`

Reply:
Ya I forget to do that.